test/fixtures/minio-fixture/src/main/java/org/elasticsearch/test/fixtures/minio/MinioTestContainer.java

```diff
@@ -48,4 +50,13 @@ public void start() {
     public String getAddress() {
         return "http://127.0.0.1:" + getMappedPort(servicePort);
     }
+
+    public void cache() {
+        try {
+            start();
```

Comment:
I'm wondering if we can be more specific here and build the image without having to actually start a container. It looks like calling `get()` on `ImageFromDockerfile` should do what we want.

Reply:
I kept it as we have the logic to which environments to skip baked into the start() method already, so not try resolving on windows etc.